internal/eval/fold_test.go

```diff
@@ -473,6 +473,26 @@ func TestFoldPolicy(t *testing.T) {
 			ast.Permit().When(ast.EntityUID("T", "1").Has("key")),
 			ast.Permit().When(ast.EntityUID("T", "1").Has("key")),
 		},
+		{
+			"opGetTag",
+			ast.Permit().When(ast.Long(42).GetTag(ast.String("key"))),
+			ast.Permit().When(ast.Long(42).GetTag(ast.String("key"))),
+		},
+		{
+			"opHasTag",
+			ast.Permit().When(ast.Long(42).HasTag(ast.String("key"))),
+			ast.Permit().When(ast.Long(42).HasTag(ast.String("key"))),
+		},
+		{
+			"opGetTagEntity",
+			ast.Permit().When(ast.EntityUID("T", "1").GetTag(ast.String("key"))),
+			ast.Permit().When(ast.EntityUID("T", "1").GetTag(ast.String("key"))),
+		},
```

Comment:
It feels like this ought to be the `opGetTag` test and the current `opGetTag` test should be `opGetTagInvalidType` test or the like.

Reply:
Yes, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/36fd3e7573bac8103b345a0c8d413d6eef37dccb.